api/restapi/configure_keptn.go

```diff
@@ -5,27 +5,24 @@ package restapi
 import (
 	"crypto/tls"
 	"fmt"
-	"github.com/keptn/keptn/api/restapi/operations/evaluation"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
 	"strings"
 
-	keptnapi "github.com/keptn/go-utils/pkg/api/utils"
-
-	"github.com/keptn/keptn/api/handlers"
-
 	"github.com/go-openapi/errors"
+	openapierrors "github.com/go-openapi/errors"
```

Comment:
not sure about this file, the header says not to be overwritten, but also this is not generated.
I can drop this is needed

Reply:
I think it's generated, but whatever :D